--- src/Makefile	2021-12-02 22:02:18.041878855 +0300
+++ src/Makefile	2021-12-02 22:03:55.685874196 +0300
@@ -7,14 +7,14 @@
 SSLCAPATH = /etc/ssl/certs
 SSLCAFILE = /etc/ssl/cert.pem
 
-MYCFLAGS =
+MYCFLAGS = `pkg-config --cflags lua5.3`
 MYLDFLAGS =
-MYLIBS =
+MYLIBS =`pkg-config --libs lua5.3`
 
 INCDIRS =
 LIBDIRS =
 
-LIBLUA = -llua
+LIBLUA =
 LIBPCRE = -lpcre2-8
 LIBSSL = -lssl
 LIBCRYPTO = -lcrypto
